library/alloc/src/collections/binary_heap.rs

```diff
@@ -1195,7 +1250,7 @@ impl<T> BinaryHeap<T> {
     /// ```
     #[inline]
     #[stable(feature = "drain", since = "1.6.0")]
-    pub fn drain(&mut self) -> Drain<'_, T> {
+    pub fn drain(&mut self) -> Drain<'_, T, A> {
         Drain { iter: self.data.drain(..) }
     }
 
```

Comment:
```suggestion
    /// Returns a reference to the underlying allocator.
    #[inline]
    #[unstable(feature = "allocator_api", issue = "32838")]
    pub fn allocator(&self) -> &A {
        self.data.allocator()
    }
```

As with https://doc.rust-lang.org/std/vec/struct.Vec.html#method.allocator